tests/server/blueprints/phenomodels/test_phenomodels_views.py

```diff
@@ -250,7 +252,7 @@ def test_phenomodel_post_add_omim_checkbox_to_subpanel(app, user_obj, institute_
     assert model_obj["subpanels"]["subpanel_x"]
 
     # GIVEN that database contains the HPO term to add to the subopanel
-    store.disease_term_collection.insert_one(omim_checkbox)
```

Comment:
Since it's no longer used it seems, why not removing the omim_checkbox fixture from tests/server/conftest.py

Reply:
It's been removed